doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -174,44 +188,48 @@ and push the changes to your fork
 GitHub pull request
 -------------------
 
-The next step is to open a GitHub pull request. The purpose of this step is
-to make your bugfix available to the community of Ceph developers.  They
-will review it and may do additional testing on it.
+The next step is to open a GitHub pull request (_PR_). This makes your bugfix
+visible to the community of Ceph contributors.  They will review it and may
+perform additional testing and / or request changes.
 
-In short, this is the point where you "go public" with your modifications.
-Psychologically, you should be prepared to receive suggestions and
-constructive criticism. Don't worry! In our experience, the Ceph project is
-a friendly place!
+This is the point where you "go public" with your modifications.  Be prepared
+to receive suggestions and constructive criticism in the form of comments
+within the PR. Don't worry!  The Ceph project is a friendly place!
 
-If you are uncertain how to use pull requests, you may read
+If you are uncertain how to create and manage pull requests, you may read
 `this GitHub pull request tutorial`_.
 
 .. _`this GitHub pull request tutorial`:
    https://help.github.com/articles/using-pull-requests/
 
-For some ideas on what constitutes a "good" pull request, see
+For ideas on what constitutes a "good" pull request, see
 the `Git Commit Good Practice`_ article at the `OpenStack Project Wiki`_.
 
 .. _`Git Commit Good Practice`: https://wiki.openstack.org/wiki/GitCommitMessages
 .. _`OpenStack Project Wiki`: https://wiki.openstack.org/wiki/Main_Page
 
+and our own :ref:`submitting-patches-to-ceph` document.
```

Comment:
```
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/dev/developer_guide/basic-workflow.rst:211: WARNING: undefined label: submitting-patches-to-ceph (if the link has no caption the label must precede a section header)
```

please note, `SubmittingPatches.rst` is not included when building the document using sphinx. so the label of `submitting-patches-to-ceph` is not visible to sphinx when building this file.

you might want to add a hyperlink pointing to an external link, specifically, `https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#submitting-patches-to-ceph`. see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks

Reply:
Sigh.  Thanks for confirming that.  Was there a way I could have known that?

I pushed an HTTPs link.  🤞

Reply:
> Was there a way I could have known that?

the source directory used by sphinx-build is `doc`, and some directory / files are excluded. you might want to check `doc/conf.py` for `exclude_patterns`. see https://www.sphinx-doc.org/en/master/usage/configuration.html for more details.